src/patternfly/components/Table/table-grid.scss

```diff
@@ -150,6 +150,10 @@ $pf-c-table-xs-breakpoint: 380px;
     }
   }
 
+  .pf-c-table__icon > * {
```

Comment:
This is another instance where I wonder if we're better off with a `__content` element in the cells that holds the content. Right now if you put 2 icons in this cell, they stack on top of one another. Or if you put text beside the icon, the text displays in the left column.

<img width="1155" alt="screen shot 2019-03-01 at 1 26 16 pm" src="https://user-images.githubusercontent.com/35148959/53661283-9b642980-3c25-11e9-9d81-56e9fd3cb192.png">


Someone would need to know to put this content in a div for things to behave as they're intended. Then we could also style `.pf-c-table__icon > .pf-c-table__cell-content { text-align: left; }`. What do you think? I think the way it is right now isn't necessarily intuitive. Also it feels kinda weird to center the icon this way. If this column is only supposed to have icons, you could use `justify-items: start;` on `.pf-c-table__icon` to align the content to the start of the grid columns.